pages/content/shared/footer.html

```diff
@@ -0,0 +1,20 @@
+$view: /layouts/blank.j2
+---
+<style amp-custom>
+  {% do doc.styles.addCssFile('css/base.css', -5) %}
+  {% do doc.styles.addCssFile('css/components/atoms/text.css', -3) %}
+  {% do doc.styles.addCssFile('css/components/atoms/anchor.css', -2) %}
+  {% do doc.styles.addCssFile('css/components/atoms/icon.css', -1) %}
+
+  {{ doc.styles.emit() }}
+</style>
```

Comment:
It would be great if this style tag could be removed.

Eventually it would be good to have a separate CSS file with that content that we could also reference and include as `amp-custom`. This is not required for now though.